docs/AboutUs.md

```diff
@@ -5,47 +5,45 @@ title: About Us
 
 We are a team based in the [School of Computing, National University of Singapore](http://www.comp.nus.edu.sg).
 
-You can reach us at the email `seer[at]comp.nus.edu.sg`
-
-## Project team
+## Project Team
 
 ### Isaac Lee
 
 <img src="images/luminousleek.png" width="200px">
 
-[[github](https://github.com/luminousleek)]
+[[GitHub](https://github.com/luminousleek)]
 
 * Role: Team Lead
 
 ### Shezad Hassan
 
 <img src="images/shezadhassan22.png" width="200px">
 
-[[github](https://github.com/shezadhassan22)]
+[[GitHub](https://github.com/shezadhassan22)]
 
 * Role: Testing
 
 ### Rohit Rajesh Bhat
 
 <img src="images/rohit0718.png" width="200px">
 
-[[github](https://github.com/rohit0718)]
+[[GitHub](https://github.com/rohit0718)]
 
 * Role: Documentation
 
-### TAN JUN RONG, BENJAMIN
+### Tan Jun Rong, Benjamin
```

Comment:
uhh want to standardised and just put Benjamin Tan?

Reply:
kk